unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarModifierAreaHUD/Scripts/AvatarModifierAreaFeedbackController.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using UnityEngine;
+
+namespace DCL.AvatarModifierAreaFeedback
+{
+    public class AvatarModifierAreaFeedbackController : IHUD
+    {
+        internal AvatarModifierAreaFeedbackView view;
+        private BaseVariable<int> InAvatarModifierAreaForSelfCounter => DataStore.i.HUDs.inAvatarModifierAreaForSelfCounter;
+        private BaseCollection<string> InAvatarModifierAreaWarningDescription => DataStore.i.HUDs.inAvatarModifierAreaWarningDescription;
+        
+        public AvatarModifierAreaFeedbackController()
+        {
+            view = AvatarModifierAreaFeedbackView.Create();
+
+            InAvatarModifierAreaForSelfCounter.OnChange += OnAvatarModifierValueChanged;
+            InAvatarModifierAreaWarningDescription.OnAdded += OnAvatarModifierWarningAdded;
+            InAvatarModifierAreaWarningDescription.OnSet += OnAvatarModifierWarningReset;
+        }
+    
+        private void OnAvatarModifierValueChanged(int current, int previous)
+        {
+            if (current == 0)
+            {
+                view.Hide();
+            }
+            if (current >= 1 && !view.isVisible)
+            {
+                view.Show();
+            }
```

Comment:
Small suggestion that I've seen being used in general in UI components, could you mode the view.Hide() and view.Show() in the SetVisibility method and call that one with the boolean condition as a parameter?

Reply:
Sound good to me. Added!